src/linear_algebra/special_linear_group.lean

```diff
@@ -209,6 +209,29 @@ subtype.ext $ (@ring_hom.map_matrix n _ _ _ _ _ _ (int.cast_ring_hom R)).map_neg
 
 end has_neg
 
+section special_cases
+
+lemma SL2_inv_det_expl (A : SL(2,R)) : det ![![A.1 1 1, -A.1 0 1], ![-A.1 1 0 , A.1 0 0]] = 1 :=
```

Comment:
I don't understand the name...

Reply:
Ok so I think what I meant to call this is `SL2_inv_expl_det` for the det of the explicit inverse. But its not a great name, I'm happy to change it to something else. I guess maybe the `SL` isn't needed...